%==============================================================================
%== template for LATEX poster =================================================
%==============================================================================
%
%--A0 beamer slide-------------------------------------------------------------
\documentclass[final]{beamer}
\usepackage[orientation=landscape,size=a0,
            scale=1.35         % font scale factor
           ]{beamerposter}
           
\geometry{
  hmargin=2.5cm, % little modification of margins
}

%
\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\hypersetup{
  colorlinks = true,
  urlcolor = blue,
}

\linespread{1.15}
%
%==The poster style============================================================
\usetheme{sharelatex}
%==Title, date and authors of the poster=======================================
\title
[BAPG XI, 6 December 2014, Davis, CA, USA] % Conference
{ % Poster title
ANGSD-wrapper: scripts to streamline and visualize NGS population genetics analysis
}

\author{ % Authors
Arun Durvasula\inst{1}, Tyler Kent\inst{1}, Siddharth Bhadra-Lobo\inst{1}, Jeffrey Ross-Ibarra\inst{2}
}
\institute
[University of California, Davis] % General University
{
\inst{1} Dept. of Plant Sciences, University of California Davis\\[0.3ex]
\inst{2} Dept. of Plant Sciences, Center for Population Biology, and Genome Center, University of California Davis\\[0.3ex]
}
\date{\today}
\begin{document}


\begin{frame}[t]
%==============================================================================
\begin{multicols}{3}
%==============================================================================
%==The poster content==========================================================
%==============================================================================

\section{Introduction}

The advent of highly multiplexed sequencing has opened a number of exciting avenues for evolutionary biologists. 
One of the powerful approaches enabled by inexpensive sequencing is the ability to sequence a large number of individuals, each to relatively low sequencing depth. 
However, this approach also presents statistical challenges in the analysis of low coverage data.  
The software ANGSD \cite{Korneliussen} and related programs \cite{Fumagalli} were developed to deal with low coverage sequence data. %Fumagalli 2014 bioinformatics http://www.ncbi.nlm.nih.gov/pubmed/24458950
Rather than call genotypes at variable sites, ANGSD performs a number of population genetic analyses on genotype likelihoods, including estimation of the population mutation rate $\theta$, the site frequency spectrum, neutrality tests, inbreeding coefficients, and population structure.
ANGSD has already been used in several studies to analyze genome sequence data \cite{Crawford} \cite{Lohmueller}. %add cites
However, ANGSD requires considerable familiarity with command line tools and remains inaccessible to many biologists that are not from a computational background. 

Here we present a software package that aids in the preparation of analyses for ANGSD and provides interactive graphing software implemented in R \cite{R} and Shiny \cite{Shiny}.  %add cites
ANGSD-wrapper simplifies multistep analyses such as calculating Tajima’s D into a single step.
Users supply all the needed information in a single configuration file (Figure \ref{configfile}), and after ANGSD has finished calculations, ANGSD-wrapper provides interactive graphing of the results (Figure \ref{shinygraph}). ANGSD-wrapper is available on github: \url{https://github.com/arundurvasula/angsd-wrapper}.

\section{Implementation}

ANGSD-wrapper is implemented using bash scripts that call ANGSD methods and handle saving intermediate files between the initial data preparation and the final data analysis. 
Each overall method in ANGSD, such as calculating estimates of $\theta$, follows a specific order of program calls. 
Thus, we have abstracted away the running of each step and provided a set of default values for parameters and instead require the user to supply the data using a configuration file (Figure \ref{configfile}). 
The user can override the default values of the parameters in the configuration file as well.
	
Additionally, ANGSD-wrapper contains a powerful graphing application based on R and Shiny (Figure \ref{shinygraph}). 
After analysis is done by ANGSD, the user can load the resulting statistics into a web-based application hosted on the user’s computer. 
This application allows the interactive plotting of values such as estimators of $\theta$ and Tajima’s D as well as the ability to load gene annotations from Ensembl. 
These features make it easy and intuitive to analyze next generation sequencing data. 

\vskip1ex
\begin{figure}[T!]
  \centering
	\includegraphics[width=0.75\columnwidth]{conf.png} 
	\caption{Example configuration file}
	\label{configfile}
\end{figure}
  
\begin{figure}[T!]
  \centering
	\includegraphics[width=0.99\columnwidth]{screenshot.png}
	\caption{Interactive plotting with Shiny showing a 200kb region along chromosome 10 of \emph{Zea mays}. Blue rectangles are annotated genes.}
	\label{shinygraph}
\end{figure}
\vskip0ex

\section{Example Application}

We sequenced 8 genomes of the wild rice \emph{Oryza glumaepatula}, 4 each from populations allopatric and sympatric to cultivated fields of the domesticated \emph{Oryza sativa} ssp. \emph{indica}, in order to investigate evidence of crop-wild introgression. 
\emph{O. glumaepatula} is a potentially endangered species 1.8 MY diverged from domesticated rice \cite{Zhang} and native to Central and South America \cite{Vaughan}. 
Both species share the AA genome \cite{Vaughan} and we have successfully crossed them experimentally, providing reason to believe the possibility of natural hybridization and thus the risk of wild population decline and extinction \cite{Rhymer}. 
Preliminary analysis using ANGSD-wrapper is suggestive of introgression (Figure \ref{gluma}): nucleotide diversity in the sympatric population of \emph{Oryza glumaepatula} is more strongly correlated with domesticated rice (Spearman's $\rho=0.537$) than with the allopatric population ($\rho=0.434$).  Nucleotide diversity in the allopatric population, in contrast, shows little correlation with diversity in domesticated samples ($\rho=0.139$).

%make this a caption on the figure
\vskip1ex
\begin{figure}
\centering
\includegraphics[width=0.99\columnwidth]{chr6rice.png}
\caption{Nucleotide diversity ($\pi$) along chromosome 6 of \emph{Oryza glumaepatula} and \emph{Oryza sativa}}
\label{gluma}
\end{figure}
\vskip0ex




%==============================================================================
%==End of content==============================================================
%==============================================================================

%--References------------------------------------------------------------------

\section{References}

\begin{thebibliography}{99}


%Buso GSG, Rangel PH and Ferreira ME (2001). Mol Ecol 7:107-117.
%Ge S, Oliveira GCX, Schaal BA, Gao LZ and Hong DY (1999) RAPD variation within and between natural populations of the 	wild rice Oryza rufipogan from China and Brazil. Heredity 82:1-7.
%Karasawa, M. M. G., et al. (2007). "Genetic structure of Brazilian wild rice (Oryza glumaepatula Steud., Poaceae) populations 	analyzed using microsatellite markers.” Genetics and Molecular Biology 30: 400-410.
%
%Oka HI (1988) Origin of Cultivated Rice. Japan Scientific Societies Press, Tokyo, and Elsevier Science Publishers, Amsterdam, 	254 pp.
%R Core Team (2014). R: A language and environment for statistical computing. R Foundation for Statistical Computing, 	Vienna, Austria. URL .
%RStudio and Inc. (2014). shiny: Web Application Framework for R. R package version 0.10.2.1. http://CRAN.R-project.org/	package=shiny
\bibitem{Korneliussen} Korneliussen et al. (2014). BMC Bioinformatics. 15:356
\bibitem{Fumagalli} Fumagalli, M., F. G. Vieira, T. Linderoth and R. Nielsen (2014). Bioinformatics 30(10): 1486-1487.
\bibitem{Crawford} Crawford, J., M. M. Riehle, W. M. Guelbeogo, A. Gneme, N. f. Sagnon, K. D. Vernick, R. Nielsen and B. P. Lazzaro (2014).  bioRxiv.
\bibitem{Lohmueller} Lohmueller, et al. (2013). American Journal of Human Genetics, 93(6), 1072–1086.
\bibitem{R} R Core Team (2014). R Foundation for Statistical Computing, Vienna, Austria \url{http://www.R-project.org/}
\bibitem{Shiny} Rstudio, Inc. (2013). \url{http://www.rstudio.com/shiny/}
\bibitem{Zhang} Zhang, Q.-J., et al. (2014). PNAS 111(46): E4954-E4962.
\bibitem{Vaughan} Vaughan DA, Morishima H, and Kadowaki K (2003). Current Opinion in Plant Biology 6:139-146.
\bibitem{Rhymer} Rhymer JM and Simberloff D (1996). Annu Rev Ecol Syst 27:83-109.

\end{thebibliography}
%--End of references-----------------------------------------------------------

\end{multicols}

%==============================================================================
\end{frame}
\end{document}
